var/spack/repos/builtin/packages/srilm/package.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+import glob
+
+
+class Srilm(MakefilePackage):
+    """SRILM (SRI Language Modeling Toolkit) is a toolkit for building
+       and applying statistical language models (LMs), primarily for
+       use in speech recognition, statistical tagging and segmentation,
+       and machine translation."""
+
+    homepage = "http://www.speech.sri.com/projects/srilm/"
+    url      = "file://{0}/srilm-1.7.3.tar.gz".format(os.getcwd())
```

Comment:
Is it likely that the software and package will support multiple versions?

If so, you may want to switch from `url` to a `def url_for_version(self, version)` approach, where the method returns a version-specific URL.

Reply:
It might but I thought Spack would be able to guess the filename as if it was a regular URL.

Reply:
> It might but I thought Spack would be able to guess the filename as if it was a regular URL.

I have just tested and it seems like it does. Why do you think `url_for_version` is needed?

Reply:
Fair enough.